### ISSUE TITLE
broken API documentation? 

### ISSUE THREAD

**Issue by jkochNU** [qutip/qutip-doc#94]
It looks like the latest API documentation has a problem (unless this is a display problem on my end only - in which case I must apologize).

**Example**: looking at http://qutip.org/docs/latest/apidoc/classes.html#qobj , what I see as the first two Parameters of Qobj is:
`inptarray_like`
`dimslist`

I suspect this should be `inpt: array_like` and `dims: list`
Is this an issue related to the space preceding the colon in the corresponding docstrings in the qutip source code?

![Capture](https://user-images.githubusercontent.com/19193849/72230547-e0622780-357b-11ea-8ede-88d8e1c22f4a.JPG)

**Comment by nathanshammah** [comment]
As you spotted, it seems related to having a space before and after the : as in [here](http://qutip.org/docs/latest/apidoc/classes.html#permutational-invariance). 

The two spaces should be a PEP style directive, and seems used also by other libraries, see NumPy [here](https://numpy.org/doc/1.17/reference/generated/numpy.dtype.html#numpy.dtype). 

It would be great to find what's messing with this. It applies only to the latest documentation (4.4).

**Comment by jkochNU** [comment]
I suspect the problem might originate from sphinx_rtd_theme. If I remove `html_theme = "sphinx_rtd_theme"`  in conf.py and build using just the default sphinx theme, then results look as expected (when including the space before the colon).

**Comment by jkochNU** [comment]
Found this thread on the github for sphinx_rtd_theme:  https://github.com/readthedocs/sphinx_rtd_theme/issues/750
There, blame was being put on incompatibilities between the rtd theme and Sphinx 2, if I understand correctly.

**Comment by jkochNU** [comment]
Helpful thread on the numpydoc github, in particular this comment: https://github.com/numpy/numpydoc/issues/215#issuecomment-529037079

For me, adding
```
.classifier:before {
    font-style: normal;
    margin: 0.2em;
    content: ":";
}
```
to custom.css solves the problem.

**Comment by jakelishman** [comment]
This is related to a `sphinx_rtd_theme`/`docutils==0.17` incompatibility.  The solution is to force version `sphinx_rtd_theme>=0.5.2` (currently only on `pip`, not `conda`), which correctly handles the dependency.  This will be fixed in the 4.6 release.

**Comment by jakelishman** [comment]
This is now fixed in the released 4.6.0 documentation release.  When we've got slightly easier methods for deploying the documentation to the website it'll be easier to "backport" fixes to previous releases, but right now our method isn't ideal so I'll leave the older versions as-is.
